include/deal.II/base/function.h

```diff
@@ -991,6 +991,20 @@ class FunctionFromFunctionObjects : public Function<dim, RangeNumberType>
                 &values,
     const double initial_time = 0.0);
 
+  /**
+   * Constructor for functions of which you only know the values.
+   *
+   * The resulting function will have a number of components equal to the size
+   * of the vector @p values. A call to the FunctionFromFunctionObject::gradient()
```

Comment:
This sentence is no longer true. Instead, you are passing the number of components explicitly.